java/lang/Float/Float.cpp

```diff
@@ -142,68 +140,483 @@ Float Float::operator/=(const Float &target) const {
     return (Float) (this->original / target.original);
 }
 
-char Float::charValue() const {
-    string convertResult = string_from_float(this->original);
-    char charValueResult = string_to_char(convertResult);
-    free(convertResult);
-    return charValueResult;
-}
-// TODO(thoangminh): enable after finish toString()
-//string Float::stringValue() const {
-//    return (string) this->toString();
-//}
-
 float Float::floatValue() const {
     return this->original;
 }
 
 short Float::shortValue() const {
-    return (short) this->original;
+    return static_cast<short> (this->original);

```

Comment:
Please use C style cast for built-in type, please check for others

Reply:
Thanks. Fixed.